scripts/install.py

```diff
@@ -246,47 +260,53 @@ def check_target_directory(target, overwrite=None):
     - ``False``: do not remove target if it exists;
     - ``True``: wipe target away if exists, without warning.
     """
-    if path.exists(target):
-        if overwrite is None:
-            print("""
-Destination directory '{target}' already exists.
-I can wipe it out in order to make a new installation,
-but this means any files in that directory, and the ones
-underneath it will be deleted.
-
-            """.format(**locals()))
-            wipe = ask(
-                "Do you want to wipe the installation directory '{target}' ?"
-                .format(** locals()))
+    if default.ANACONDA:
+        if os.path.basename(os.path.normpath(target)) in check_output(['conda', 'env', 'list']):
+            wipe = ask("Do you want to wipe the existing conda environment {e}?".format(e=os.path.basename(os.path.normpath(target))))
             if wipe:
-                overwrite = True
+                check_call(['conda', 'remove', '-n', os.path.basename(os.path.normpath(target)), '--all', '-y'])
             else:
-                overwrite = False
+                die(os.EX_CANTCREAT,
+                    "Unable to create virtual environment: target directory already exists",
+                    """
+                        The script was unable to create an Anacoda virtual environment because it already exists.
+
+                        In order to proceed, you must take the following action:
 
-        # if directory exists and cannot be overwritten, abort
-        if not overwrite:
+                        * run this script again adding '--overwrite' option, which will
+                        overwrite the {target} directory, or
+                    """.format(**locals()))
+
+    if path.exists(target):
+        print("""
+            Destination directory '{target}' already exists.
+            I can wipe it out in order to make a new installation,
+            but this means any files in that directory, and the ones
+            underneath it will be deleted.
+            """.format(**locals()))
+        wipe = ask(
+            "Do you want to wipe the installation directory '{target}' ?".format(**locals()))
+        if wipe:
+            logging.info("Deleting directory '%s' as requested ...", target)
+            shutil.rmtree(target)
+        else:
             die(os.EX_CANTCREAT,
-                "Unable to create virtualenv: target directory already exists",
+                "Unable to create virtual environment: target directory already exists",
                 """
-The script was unable to create a virtual environment in "{target}"
-because the directory already exists.
+                    The script was unable to create a virtual environment in "{target}"
```

Comment:
Lines 296--307: text to be printed as script output should not be indented (like for lines 140--163)